lib/connect_proto/src/order.proto

```diff
@@ -95,7 +100,7 @@ message Order {
     google.protobuf.Timestamp completion_date_time = 6; // Timestamp when the results were composed into a report and released.
     string expiration_date = 7; // YYYY-MM-DD, Date when the order becomes invalid
     Specimen specimen = 8;
-    CodedValue procedure = 9; // Deprecating in favor of procedures (25)
+    Procedure procedure = 9; // Deprecating in favor of procedures (25)
```

Comment:
I think maybe we should keep this one "as-is", since it's deprecated and we don't want to break CicHealth.